src/Sulu/Bundle/AudienceTargetingBundle/Tests/Functional/CachingTest.php

```diff
@@ -38,6 +38,7 @@ public function testFirstRequestIsACacheMiss()
         $cacheKernel = new AppCache(self::bootKernel());
         $cookieJar = new CookieJar();
         $client = new KernelBrowser($cacheKernel, [], null, $cookieJar);
+        $client->disableReboot();
```

Comment:
this is now required else other tests are running into an issue about a not booted kernel